src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -379,7 +379,9 @@ private static bool IsFloatingPointExpressionOfUnknownPrecision(BoundExpression
                     {
                         var mg = (BoundMethodGroup)rewrittenOperand;
                         Debug.Assert(oldNodeOpt.SymbolOpt is { });
-                        return new BoundFunctionPointerLoad(oldNodeOpt.Syntax, oldNodeOpt.SymbolOpt, type: funcPtrType, hasErrors: false);
+                        return new BoundFunctionPointerLoad(oldNodeOpt.Syntax, oldNodeOpt.SymbolOpt,
```

Comment:
If we made it to lowering with a `static abstract` method group without a type parameter receiver, does that mean we failed to give a diagnostic during binding?

Reply:
> If we made it to lowering with a static abstract method group without a type parameter receiver, does that mean we failed to give a diagnostic during binding?

Pretty much.